contributors/design-proposals/scheduling/scheduling-policy.md

```diff
@@ -0,0 +1,781 @@
+spec:
+  required:
+    nodeSelectors:
+      beta.kubernetes.io/arch: ["amd64", "arm64"]
+    priorityClasseNames: ["bronze", "gold", "silver"]
+  allowed:
+    nodeSelectors:
+      disk: ["ssd"]
+  default:
+    nodeSelector:
+      beta.kubernetes.io/arch: "amd64"
+    priorityClasseName: "bronze"
+```
+## Default SchedulingPolicies
+
+### Restricted policy
+Here is a reasonable policy that might be allowed for any cluster without specific needs:
+
+```yaml
+apiVersion: extensions/valpha1
+kind: SchedulingPolicy
+metadata:
+  name: restricted
+spec:
+  allowed:
+    schedulerNames: ["default-scheduler"]
+```
+
+It only allows usage of the default scheduler, no tolerations, nodeSelectors nor affinities.
+
+Multi-archi (x86_64, arm) or multi-OS (Linux, Windows) clusters might also allow the following nodeSelectors:
+
+```yaml
+apiVersion: extensions/valpha1
+kind: SchedulingPolicy
+metadata:
+  name: restricted-multiarch-by-node-selector
+spec:
+  required:
+    schedulerNames: ["default-scheduler"]
+    nodeSelectors:
+      beta.kubernetes.io/arch: ["amd64", "arm64"] # pick one in required values
+      beta.kubernetes.io/os: ["Linux", "Windows"] # pick one
+  default: # if not set, inject to pods definitions
+    schedulerName: "default-scheduler"
+    nodeSelector:
+      beta.kubernetes.io/arch: "amd64"
+      beta.kubernetes.io/os: "Linux"
+```
+
+```yaml
+apiVersion: extensions/valpha1
+kind: SchedulingPolicy
+metadata:
+  name: restricted-multiarch-by-affinity
+spec:
+  required:
+    schedulerNames: ["default-scheduler"]
+    affinities:
+      nodeAffinities:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - keys: ["beta.kubernetes.io/arch"]
+              operators: ["In"]
+              values: ["amd64", "arm64"]
+          - matchExpressions:
+            - keys: ["beta.kubernetes.io/os"]
+              operators: ["In"]
+              values: ["Linux", "Windows"]
+  default: # if not set, inject to pods definitions
+    schedulerName: "default-scheduler"
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - key: "beta.kubernetes.io/arch"
+              operator: "In"
+              values: ["amd64"]
+          - matchExpressions:
+            - key: "beta.kubernetes.io/os"
+              operator: "In"
+              values: ["Linux"]
+```
+
+### Privileged Policy
+
+This is the privileged SchedulingPolicy, it allows usage of all schedulers, priority classes, nodeSelectors, affinities and tolerations.
+
+```yaml
+apiVersion: extensions/valpha1
+kind: SchedulingPolicy
+metadata:
+  name: privileged
+spec:
+  allowed:
+    schedulerNames: []
+    priorityClasseNames: []
+    nodeSelectors: {}
+    tolerations: []
+    affinities: {}
+```
+
+## RBAC
+SchedulingPolicy are supposed to be allowed using the verb `use` to apply at pod runtime
```

Comment:
I strongly discourage this approach. See [Policy Binding](https://github.com/tallclair/k8s-community/blob/pod-restrict/contributors/design-proposals/auth/pod-restriction.md#policy-binding)

Reply:
@tallclair @bsalamat @liggitt @smarterclayton - I'm fine with having a list of namespaces + namespace selector.